v21.2/performance-best-practices-overview.md

```diff
@@ -318,15 +318,17 @@ However, because `AS OF SYSTEM TIME` returns historical data, your reads might b
 
 ## Hot spots
 
-Transactions that operate on the same range but _different index keys_ are limited by the overall hardware capacity of [the range lease holder](architecture/overview.html#terms) node. These are referred to as _hot spots_.
+A *hot spot* is any location on the cluster, such as a range, receiving significantly more requests than another. Hot spots can cause problems as requests increase.
 
-Hot spots can occur when a range is indexed on a column of data that is sequential in nature such that all incoming writes to the range will be the last (or first) item in the index and appended to the end of the range. As a result, the system cannot find a point in the range that evenly divides the traffic, and the range cannot benefit from [load-based splitting](load-based-splitting.html), creating a hot spot on the single range.
+They commonly occur with transactions that operate on the **same range but different index keys**, which are limited by the overall hardware capacity of [the range leaseholder](architecture/overview.html#terms) node.
+
+Hot spots can occur when a range is indexed on a column of data that is sequential in nature (e.g., [an ordered sequence](sql-faqs.html#what-are-the-differences-between-uuid-sequences-and-unique_rowid), or a series of increasing, non-repeating [`TIMESTAMP`s](timestamp.html)), such that all incoming writes to the range will be the last (or first) item in the index and appended to the end of the range. As a result, the system cannot find a point in the range that evenly divides the traffic, and the range cannot benefit from [load-based splitting](load-based-splitting.html), creating a hot spot at the single range.
```

Comment:
This wasn't something you changed btw....

Is the range not being able to benefit from load-based splitting _a consequence of_ the system not finding a point in the range that evenly divides the traffic? 

I wasn't sure if the load-based splitting clause was an aside adding to the reasons why a hot spot can't be created, or:

"As a result, the system cannot find a point in the range that evenly divides the traffic, **so** the range cannot benefit from [load-based splitting](load-based-splitting.html)....."

That's not necessarily a suggestion, I just was trying to understand the connection of each part of the sentence.